### PR TITLE
Fix: renamed singIn methods to signIn

### PR DESCRIPTION
**Pull request by danielgary**
Changed:
- [x] Renamed singIn methods to signIn

### CODE REVIEW

File: yarn-error.log [hunk 1]
```diff
@@ -0,0 +1,234 @@
+Arguments:
```
Comment (kentcdodds) [inline yarn-error.log]: Could you get rid of this please?

**Review by kentcdodds — COMMENTED** [review]
Nice! Thanks. One thing.

File: yarn-error.log (hunk 1 above)
Comment (danielgary) [reply yarn-error.log]: Already gone

**Comment by kentcdodds** [comment]
Sweet! Thanks!

**Pull request merged**